src/main/java/seedu/address/logic/commands/UntagCommand.java

```diff
@@ -101,5 +101,25 @@ public CommandResult removeUserTags(Model model) throws CommandException {
                 + "Modules: " + editedUser.getImmutableModuleTags().toString()));
     }
 
+    public ContactIndex getIndex() {
+        return this.index;
+    }
+
+    public Set<ModuleTag> getModules() {
+        return this.moduleTags;
+    }
```

Comment:
Just checking: the`getModules()` and `getGroups()` will be two separate methods?

Reply:
Likely will be extending tag to do groups as well so likely will be two separate methods!

Reply:
Wow! What a good plan!

Reply:
Agreed